Source/Utility/looper.h

```diff
@@ -0,0 +1,287 @@
+#pragma once
+#include <cmath>
```

Comment:
dsp.h should include the std cpp math lib. So I don't think there's any need to keep cmath included here.